pegasus/sites.v3/code.org/public/markdown/more.md

```diff
@@ -89,6 +89,13 @@ Cu vis minim imperdiet. Vel cu phaedrum adolescens, vim te assentior delicatissi
 
 
 ## Social Media
+
+### How to point to Code.org's social media sites
+Do not use any Facebook, Twitter, etc. widgets that connect users to like or follow Code.org on social media. These widgets use cookies, which violates our privacy policy. Instead, you can show an icon or image of the social media platform's logo and link to Code.org's social media accounts.
```

Comment:
Might be worth explaining that the difference is that we don't want to embed any content hosted on a third party site.  If we include an icon or image from that third-party site they'll also be able to track.

Reply:
What do you mean include an icon or image from another site? Anything other than FontAwesome?

Reply:
Any time we include an asset from another site, it gives them a chance to track that user's activity.  So in this case the message is that we want to host the social media platform's logo/button on our own site.  We don't want to embed anything that we are not hosting when it comes to these social media assets.

Reply:
This more clear?








`Do not use any Facebook, Twitter, etc. widgets that connect users to like or follow Code.org on social media. These widgets use cookies that track our users, which violates our privacy policy. Instead, you can show an icon or image of the social media platform's logo and link to Code.org's social media accounts. By using this method, we host the social media platform's logo/button on our own site.`